packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { number, string, bool, shape } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import moment from 'moment';
```

Comment:
What's the reason for changing this from `moment-timezone` to `moment`?

Reply:
I don't see the point to import the `timezone` here when is imported and used in the formatting functions.